src/main/java/com/game/Game.java

```diff
@@ -0,0 +1,201 @@
+package com.game;
+
+import com.controller.ViewFactory;
+import com.controller.WindowType;
+import com.model.RecordManager;
+import javafx.animation.AnimationTimer;
+import javafx.application.Platform;
+import javafx.scene.Group;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.control.TextInputDialog;
+import javafx.scene.image.ImageView;
+import javafx.scene.input.KeyCode;
+import javafx.scene.layout.Pane;
+import javafx.stage.Stage;
+
+import java.util.*;
+
+enum SimpleDirection {
```

Comment:
Consider creating global enum in separate file so I can be reused in different classes.